### PR TITLE
Add several spelling corrections + refinements

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -25642,6 +25656,10 @@ skeep->skip
 skelton->skeleton
 skept->skipped
 sketchs->sketches
+sketck->skate, sketch,
+sketcked->skated, sketched,
+sketckes->skates, sketches,
+sketcking->skating, sketching,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:25662]: Swap these round as the second term is clearly the most likely one.

Comment (peternewman) [reply codespell_lib/data/dictionary.txt:25662]: In fact I can't really see how the first term is likely in any of these cases...

**Review by peternewman — CHANGES_REQUESTED** [review]

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:25662]: I'll swap them but i think we should keep both. I think I saw the original typo somewhere where the correct word was `sketck`.

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 79a4879

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: As in `sketck->skate`?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -24637,6 +24650,7 @@ rmove->remove
 rmoved->removed
 rmoving->removing
 rnage->rage, range,
+ro->to, row,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:24653]: Also rob, rod, roe and rot

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: yes. that is what I meant

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Okay fair enough.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (bl-ue) [reply codespell_lib/data/dictionary.txt:24653]:
```suggestion
ro->to, row, rob, rod, roe, rot,
```

**Review by bl-ue — APPROVED** [review]

**Pull request merged**